src/noise-filter.js

```diff
@@ -46,31 +97,100 @@ export default class NoiseFilter {
     }
 
     update = ext => {
-        const loudness = ext.getAvg('loudness', loudness => loudness.total)
-        const strength = logMap(loudness, 0, 24, 1, 15)
 
-        const sharpness = ext.getAvg('perceptualSharpness')
-        const baseRoughness = logMap(sharpness, 0, 1, 0, 2)
+        var settingsToUpdate = {}
+
+        for (var o in this.mods){
+            var curo = this.mods[o]
+            if (curo.mapTo){
+                var newVal
+                if (curo.mapTo == 'loudness') newVal = ext.getAvg('loudness', loudness => loudness.total);
+                else newVal = ext.getAvg(`${curo.mapTo}`)
```

Comment:
No need for template string:
`else newVal = ext.getAvg(curo.mapTo)`

Reply:
thx will change that